Converter/igtlioVideoConverter.cxx

```diff
@@ -41,9 +38,28 @@ namespace igtlio
       // TODO: error handling
       return 0;
     }
+    GenericDecoder* decoder = NULL;
+#if defined(USE_H264)
+  if(videoMsg->GetCodecType().compare(CodecNameForH264)==0)
```

Comment:
This a case sensitive compare, is this ok?

Reply:
Also, what is the OpenIGTLinkIO coding convention for constants?

Reply:
Thanks for point out.  I think the case sensitive comparison is fine. As the videomessage has a SetCodecType function which will takes the codec name constants. probably in the function i should check if the use input codec name is valid and return error code if not valid.

The constants are defined actually in OpenIGTLink Library. In IGTLInk library, the constants should be like "IGTL_VIDEO_ENDIAN_LITTLE" all capital. Will modify the constants accordingly